config/routes.rb

```diff
@@ -227,6 +227,7 @@
   get 'feed/liked' => 'notes#liked_rss'
 
   get 'dashboard' => 'home#dashboard'
+  get 'dashboard2' => 'home#dashboard2'
```

Comment:
`v2/dashboard` ?
USing v2 we can document also what was missing in v1 maybe, what do you think?

Reply:
Yes, this makes sense! I can use v2